src/type.ts

```diff
@@ -0,0 +1,3 @@
+export interface FlagsObjectInterface {
+    [key: string]: string
```

Comment:
The type should be `[key:string]: any`, 

Think of command like
`node index.js -f <FILE>`
`node index.js -f <OBJECT>`

While reading from CLI, objects will be stringified, but we need to return it as an object literal so consumers can use it.

Hence, `[key:string]:any` is preferred.